lib/twilio/audiohelper.ts

```diff
@@ -1024,6 +1033,11 @@ namespace AudioHelper {
      */
     enumerateDevices?: any;
 
+    /**
+     * Promise to wait for {@link Device.connect}.
+     */
+    getMakeCallPromise: () => Promise<any>;
+
```

Comment:
Rename and make optional.
```suggestion
    /**
     * Promise to wait for before setting the input device.
     */
    beforeSetInputDevice?: () => Promise<any>;

```